plugins/survey-assets/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@hcengineering/survey-assets",
+  "version": "0.0.1",
+  "main": "src/index.ts",
+  "author": "Anticrm Platform Contributors",
```

Comment:
Please use `Hardcore Engineering Inc` for new modules.

Reply:
Changed